workflow_examples/mc_workflow.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name":"mc_workflow",
+    "num_jobs": 60,
+    "infiles_per_job":1,
```

Comment:
Same here, type is `calculation`, so why bother to define `infiles_per_job`?

Reply:
See above

Reply:
see above

Reply:
Done: https://github.com/HEPCompSim/DCSim/pull/41/commits/872bf168f39d3886a58a436810d4d335590a9569